websites/H/HTB Academy/presence.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-one-time-vars/no-one-time-vars */
+const presence = new Presence({
+		clientId: "1200517025383075840",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	{ pathname } = document.location;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://images.g2crowd.com/uploads/product/image/large_detail/large_detail_fabbd1201be6463ecfda11b3bbcb03de/hack-the-box-for-business.jpg",
+		startTimestamp: browsingTimestamp,
+	};
+
+	if (pathname === "/" || pathname === "/login")
+		presenceData.details = "Breaching into the Academy";
+	else if (pathname.includes("/dashboard")) {
+		presenceData.details = "Browsing the dashboard";
+		const persred: HTMLElement = document.querySelector(".red .progress"),
+			persblue: HTMLElement = document.querySelector(".blue .progress"),
+			persgrn: HTMLElement = document.querySelector(".green .progress");
+		presenceData.state =
+			`Off: ${persred.textContent} ` +
+			`Def: ${persblue.textContent} ` +
+			`Gen: ${persgrn.textContent}`;
+	} else if (pathname.includes("/exams"))
+		presenceData.details = "Browsing the exams";
+	else if (pathname.includes("/paths")) presenceData.details = "Browsing paths";
+	else if (pathname.includes("/modules"))
+		presenceData.details = "Browsing modules";
+	else if (pathname.includes("/section")) {
+		let module: HTMLElement;
+		const iterminal: HTMLElement = document.querySelector(".iterminal");
+		if (iterminal) {
+			module = document.querySelector(
+				"#layout-wrapper > div.main-content > div > div:nth-child(2) > div > div > h4"
+			);
+		} else {
+			module = document.querySelector(
+				"#layout-wrapper > div.main-content > div > div:nth-child(1) > div > div > h4"
+			);
+		}
```

Comment:
If possible, see if these selectors can be simplified. For example, `h4` or `.main-content h4`, etc.

Reply:
This one is not clearly possible. I've tried it while other change request. I'll give it another try though.